app/components/cases/new-subcase.js

```diff
@@ -80,7 +80,7 @@ export default class CasesNewSubcase extends Component {
       type: this.type,
       shortTitle: trimText(this.shortTitle),
       title: trimText(this.title),
-      confidential: this.args.case.confidential,
+      confidential: false,  // Set default to false as it isn't derived from case anymore
```

Comment:
`it isn't derived from case anymore` suggests that the property still exists on case, but isn't taken in to account here. I would suggest to remove this comment.

Reply:
Strange that you still saw that comment, as I removed it a previous commit
https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/1119/commits/deccd6edf2c76ea607eb21035f0ac7a9e4fc060f

Reply:
But the code will change again, since there will be a toggle added to the modal so you can create a confidential subcase instead of having to change it right after creating